pkg/sql/catalog/dbdesc/database_test.go

```diff
@@ -275,7 +275,7 @@ func TestValidateCrossDatabaseReferences(t *testing.T) {
 			cb.UpsertDescriptorEntry(schemadesc.NewBuilder(&schemaDesc).BuildImmutable())
 		}
 		_ = cb.ForEachDescriptorEntry(func(desc catalog.Descriptor) error {
-			cb.UpsertNamespaceEntry(desc, desc.GetID())
+			cb.UpsertNamespaceEntry(desc, desc.GetID(), desc.GetModificationTime())
```

Comment:
The namespace entries in an `nstree.Catalog` are now `nstree.NamespaceEntry` which extends `catalog.NameEntry` to include MVCC timestamps. This was necessary because it allowed pushing the last remaining kv-level table scan in `descs` into `catkv` and rewrite it using the `catalogQuerier` machinery. 

Also in one use case we do care about these timestamps, and they were pretty straightforward to add, so 🤷‍♂️

Reply:
100% agree. We'll want to be principled about timestamps at some point.